### ISSUE TITLE
Seamlessly embedded tables are narrow

### ISSUE THREAD

**Issue by pointlessone** [kepano/obsidian-minimal#498]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.1.9

### Describe the bug

Embedded tables do not take on `table-wide` utility class.

Additionally, embedded DataView tables are even narrower than regular embedded tables (even with `embed-strict`).

Here's an illustration:

![Screenshot 2022-11-22 at 11 22 03](https://user-images.githubusercontent.com/7405/203281541-fd31e15d-b5d0-4b0f-b914-27ecbdcdb577.png)

I added guides because with all the censorship and low contrast it's unclear what's going on.

🟥 Red is the extent of the wide table in this note (produced by DataView)
🟩 Green is the extent of the node body
🟨 Yellow is the extent of the embedded table (produced by DataView)

At the top of the image is a DataView table in this note. It is, as expected, wide. In the middle is an embedded DataView table. It's narrower even than the note body. At the bottom is an embedded regular table. It's exactly as wide as the note body. This note has `embed-strict` and `table-wide` classes.

<details><summary>Debug info</summary>

```
SYSTEM INFO:
	Obsidian version: v1.0.3
	Installer version: v0.15.9
	Operating system: Darwin Kernel Version 22.2.0: Tue Nov  8 21:14:46 PST 2022; root:xnu-8792.60.55.131.1~1/RELEASE_X86_64 22.2.0
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: off
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal
	Snippets enabled: 3
	Restricted mode: off
	Plugins installed: 35
	Plugins enabled: 31
		1: Smart Random Note v0.2.1
		2: Periodic Notes v0.0.17
		3: Obsidian Query Language v2.1.1
		4: Note Refactor v1.7.1
		5: Natural Language Dates v0.6.1
		6: macOS Keyboard Navigation v1.1.2
		7: Format Hotkeys v0.1.7
		8: Footnote Shortcut v0.0.9
		9: Folder Note v0.7.3
		10: Dataview v0.5.47
		11: Calendar v1.5.10
		12: Better Word Count v0.8.1
		13: Advanced Tables v0.17.3
		14: Find orphaned files and broken links v1.8.0
		15: Templater v1.16.0
		16: Advanced Appearance v1.1.0
		17: Gallery v0.5.8
		18: Admonition v9.0.5
		19: Key Promoter v1.2.2
		20: Supercharged Links v0.9.3
		21: Reminder v1.1.14
		22: Minimal Theme Settings v6.1.0
		23: Contextual Typography v2.2.4
		24: Hider v1.2.3
		25: Editor Syntax Highlight v0.1.3
		26: CodeMirror Options v0.9.5
		27: Style Settings v0.4.12
		28: Heatmap Calendar v0.5.2
		29: Tasks v1.18.1
		30: Snippet Downloader v2.0.1
		31: TikZJax v0.5.1

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

</details>

**Comment by geekyorganizer** [comment]
Same issue, but only in Reading view. In Live Preview the table width is displayed correctly.

**Comment by kepano** [comment]
Should be fixed in 7.0
